pkgs/development/python-modules/wasmer/default.nix

```diff
@@ -20,7 +20,12 @@ let
       sha256 = "0302lcfjlw7nz18nf86z6swhhpp1qnpwcsm2fj4avl22rsv0h78j";
     };
 
-    cargoSha256 = "0d83dniijjq8rc4fcwj6ja5x4hxh187afnqfd8c9fzb8nx909a0v";
+    cargoSha256 = {
```

Comment:
does cargo, depending on python version, fetch different dependencies?

Reply:
another possibility is that there's some reference to the python version, thus the vendor libs get a different hash

Reply:
The problem is that the Python version is used in the `name` attribute of the Rust derivation. `name` is used as part of the vendor directory and tarball. So, if the Python version is part of the name, the vendored tarball changes for every Python version.

Fixed in #110580.

Reply:
Going to close this in favor of daniels PR.